.travis.yml

```diff
@@ -1,16 +1,30 @@
 sudo: false
 language: d
 d:
-  - dmd-nightly
-  - dmd-beta
-  - dmd
-  - ldc-beta
-  - ldc
+- dmd-nightly
```

Comment:
In case someone wonders about this. I reformated the entire file with the official Travis CLI (this was needed to insert the `secret`).